compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/test/insert_slice.mlir

```diff
@@ -68,3 +69,42 @@ func.func @insert_slice_within_dispatch_workgroups_not_converted() -> tensor<f32
   }
   return %0 : tensor<f32>
 }
+
+// -----
+
+func.func @insert_slice_convert_dynamic_offset_and_size
+    (%arg0 : tensor<?x24x48xf32>, %arg1 : tensor<1x?x48xf32>, %offset : index, %size : index) ->
```

Comment:
function style is (%arg0: tensor<..>, %arg1: tensor<...>) (no spaces between arg name and type)